drivers/display/display_stm32_ltdc.c

```diff
@@ -439,7 +473,9 @@ static const struct display_driver_api stm32_ltdc_display_api = {
 	.read = stm32_ltdc_read,
 	.get_capabilities = stm32_ltdc_get_capabilities,
 	.set_pixel_format = stm32_ltdc_set_pixel_format,
-	.set_orientation = stm32_ltdc_set_orientation
+	.set_orientation = stm32_ltdc_set_orientation,
+	.blanking_off = stm32_ltdc_display_blanking_off,
+	.blanking_on = stm32_ltdc_display_blanking_on
```

Comment:
nit: keep the `,` on the last element, then the next change won't have the stray diff